src/index.html

```diff
@@ -1,159 +1,169 @@
-              <td class="activity three-day-activity"></td>
-              <td class="activity four-day-activity"></td>
-              <td class="activity five-day-activity"></td>
-              <td class="activity six-day-activity"></td>
+              <th class="weekly-sleep">Next day</th>
+              <td class="sleep yesterday-sleep"></td>
             </tr>
             <tr>
-              <th class="weekly-activity">Today</th>
-              <th class="weekly-activity">Yesterday</th>
-              <th class="weekly-activity">2 Days Ago</th>
-              <th class="weekly-activity">3 Days Ago</th>
-              <th class="weekly-activity">4 Days Ago</th>
-              <th class="weekly-activity">5 Days Ago</th>
-              <th class="weekly-activity">6 Days Ago</th>
+              <th class="weekly-sleep">2 days later</th>
+              <td class="sleep two-day-sleep"></td>
             </tr>
-          </table>
-        </div>
-        <div class="sleep-stats reset hidden">
-          <h2>Today's Sleep</h2>
-          <p class="today-sleep"></p>
-          <div class="avg-sleep-stats">
-          </div>
-          <table class="sleep-data table">
             <tr>
-              <td class="sleep today-sleep"></td>
-              <td class="sleep yesterday-sleep"></td>
-              <td class="sleep two-day-sleep"></td>
+              <th class="weekly-sleep">3 days later</th>
               <td class="sleep three-day-sleep"></td>
+            </tr>
+            <tr>
+              <th class="weekly-sleep">4 days later</th>
               <td class="sleep four-day-sleep"></td>
-              <td class="sleep five-day-sleep"></td>
-              <td class="sleep six-day-sleep"></td>
             </tr>
             <tr>
-              <th class="weekly-sleep">Today</th>
-              <th class="weekly-sleep">Yesterday</th>
-              <th class="weekly-sleep">2 Days Ago</th>
-              <th class="weekly-sleep">3 Days Ago</th>
-              <th class="weekly-sleep">4 Days Ago</th>
-              <th class="weekly-sleep">5 Days Ago</th>
-              <th class="weekly-sleep">6 Days Ago</th>
+              <th class="weekly-sleep">5 days later</th>
+              <td class="sleep five-day-sleep"></td>
             </tr>
-            </table>
-        </div>
-        <div class="hydration-stats reset hidden">
-          <h2>Today's Water Consumption</h2>
-          <p class="today-consumption"></p>
-          <table class="water-data table">
             <tr>
+              <th class="weekly-sleep">6 days later</th>
+              <td class="sleep six-day-sleep"></td>
+            </tr>
+              </table>
+          </div>
+          <div class="hydration-stats reset hidden">
+            <h2>Today's Water Consumption</h2>
+            <p class="today-consumption"></p>
+            <table class="water-data table">
+              <th class="weekly-water">Start date</th>
```

Comment:
If the above mentioned comments needs a `tr`, this does as well.

Reply:
added